Transition to targets (staying on separate branch for now) (#65)
USGS-R/ds-pipelines-1-course: responses/20a-watch-targets.md

Great comments @{{ user.username }}! :sparkles:

> You could consider [GNU make](https://www.gnu.org/software/make/) to be a great grandparent of the packages we referred to early in this lesson (`remake`, `scipiper`, `drake`, and `targets`). The lead of `targets` has added a lot of useful features to dependency management systems in R, and has a great way of summarizing why we put energy into using these tools. **"Skip the work you don't need"**

Reviewer: ```suggestion
You could consider [GNU make](https://www.gnu.org/software/make/) to be a great grandparent of the packages we referred to early in this lesson (`remake`, `scipiper`, `drake`, and `targets`). Will Landau, the lead developer of `targets`, has added a lot of useful features to dependency management systems in R, and has a great way of summarizing why we put energy into using these tools: **"Skip the work you don't need"**
```

We'd like you to next check out a short part of Will's video on `targets`

[![reproducible workflows with R targets](https://user-images.githubusercontent.com/2349007/81425244-cea12300-911c-11ea-8555-0c5a61fbee21.png)](https://www.youtube.com/watch?v=Gqn7Xn4d5NI "Will Landau, Reproducible computation at scale in R with targets, NYC Data Hackers virtual meetup")

### :tv: Activity: watch video on targets from at least 7:20 to 11:05 (you are welcome to watch the full main talk from 7:20 to 7:40)

> Use a github comment on this issue to let us know what contrasts you identified between solutions in `make` and what is offered in R-specific tools, like `targets`. Please use less than 300 words. Then assign your onboarding cohort team member this issue to read what you wrote. When they are done, welcome them to comment with any questions or comments.   

Reviewer: Does this new video clip also talk about `make`?

And a suggested change to the current language:
```suggestion
Use a github comment on this issue to let us know what contrasts you identified between solutions in `make` and what is offered in R-specific tools, like `targets`. Please use less than 300 words. Then assign your onboarding cohort team member this issue to read what you wrote and respond with any questions or comments.   
```

Author: I think this part is referring to the previous step which has the user watch [this video](https://www.youtube.com/watch?v=u-ZgiYjJ4lA) by Karl Broman all about make.

Reviewer: Oh! Perfect. I think I accidentally skipped over some of the pre-existing content as I was reviewing.
